elements/rh-secondary-nav/demo/rh-secondary-nav.html

```diff
@@ -173,11 +162,11 @@ <h2 slot="header">
         </rh-secondary-nav-menu>
       </rh-secondary-nav-dropdown>
      </li>
-    <li><a href="#">Documentation</a></li>
-    <li><a href="#">Extra Placeholder</a></li>
+    <li><a href="#" data-analytics-level="1" data-analytics-text="Documentation">Documentation</a></li>
```

Comment:
Also need to verify with Marcia & Rickey if a *-category attr is required at this level.

Reply:
Level 1 elements in the main nav don't have category attributes so followed that structure here.